script.js

```diff
@@ -0,0 +1,841 @@
+const FUNCTIONAL_KEYS = {
+  shift: false,
+  shiftRight: false,
+  ctrl: false,
+  ctrlRight: false,
+  alt: false,
+  altRight: false,
+  windows: false,
+  caps: false,
+};
+
+if (!window.localStorage.getItem("virtualKeyBoardLang")) window.localStorage.setItem("virtualKeyBoardLang", "en");

```

Comment:
It's really hard to read such conditions. Better
`if(your condition) {    

    code here

}`

Reply:
Moved these lines to class and refactored
https://github.com/BlackTruth/virtual-keyboard/blob/6297a0817c03affb1948c9e73a6352687e6ff449/src/classes/VirtualKeyboard.js#L21-L24